fictadvisor-web/src/components/pages/account-page/components/group-tab/components/table/student-table/StudentsTable.styles.ts

```diff
@@ -22,15 +25,27 @@ export const tag: SxProps<Theme> = {
 };
 
 export const iconButton = (isMobile: boolean) => ({
-  transform: !isMobile ? 'scale(1.41)' : 'scale(1)',
+  transform: !isMobile ? 'scale(1.41)' : 'scale(1.15)',
   aspectRatio: '1/1',
   alignSelf: 'center',
+  margin: !isMobile ? '0px 0px 0px 8px' : '0px',
```

Comment:
why such long value. if we need only margin left then write it as property and get rid of long values